ttf-arphic-uming.yaml

```
@@ -0,0 +1,33 @@
+package:
+  name: ttf-arphic-uming
+  version: 0.2.20080216.2
+  epoch: 0
+  description: CJK Unicode font Ming style
+  copyright:
+    - license: Arphic Public License
```

```suggestion
    - license: Arphic-1999
```

https://spdx.org/licenses/Arphic-1999.html